### ISSUE TITLE
Warning: De: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.

### ISSUE THREAD

**Issue by mukurpuri** [reachtokish/rctx-contextmenu#51]
- `rctx-contextmenu` version: ^1.4.1
- `node` version: v20.13.1
- `npm` (or `yarn`) version: 10.5.2

Vite - ^5.3.1

Using 
```import { ContextMenu, ContextMenuItem, ContextMenuTrigger } from 'rctx-contextmenu';```?

is giving warnings in console 
**chunk-RY6NLCXT.js?v=9a219bf8:521 Warning: Me: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.
    at Me (http://localhost:5173/node_modules/.vite/deps/rctx-contextmenu.js?t=1722055804221&v=9a219bf8:797:15)**
    
    
<img width="1006" alt="image" src="https://github.com/user-attachments/assets/523c0c07-fecf-4d7a-b534-683465766f60">

**Comment by caironite** [comment]
React v19 breaks the component altogether.

**Comment by ektak26** [comment]
is there any solution? I am using react 18.3.1 and rctx-contextmenu : 1.4.1

**Comment by caironite** [comment]
Yes. I forked the project and made some fixes to make it work on React 19.

**Comment by caironite** [comment]
Here's the Github project:
https://github.com/caironite/react-context-menu-19

**Comment by caironite** [comment]
https://www.npmjs.com/package/react-context-menu-19
